pkg/server/config.go

```diff
@@ -138,20 +138,21 @@ type InternalEtcdConfig struct {
 }
 
 type CellsConfig struct {
-	BootImageSpec         cloud.BootImageSpec           `json:"bootImageSpec"`
-	DefaultInstanceType   string                        `json:"defaultInstanceType"`
-	DefaultVolumeSize     string                        `json:"defaultVolumeSize"`
-	StandbyCells          []nodemanager.StandbyNodeSpec `json:"standbyCells"`
-	CloudInitFile         string                        `json:"cloudInitFile"`
-	Itzo                  ItzoConfig                    `json:"itzo"`
-	ExtraCIDRs            []string                      `json:"extraCIDRs"`
-	ExtraSecurityGroups   []string                      `json:"extraSecurityGroups"`
-	Nametag               string                        `json:"nametag"`
-	StatusInterval        int                           `json:"statusInterval"`
-	HealthCheck           HealthCheckConfig             `json:"healthcheck"`
-	PrivateIPOnly         *bool                         `json:"privateIPOnly"`
-	CellConfig            map[string]string             `json:"cellConfig"`
-	DefaultIAMPermissions string                        `json:"defaultIAMPermissions"`
+	BootImageSpec          cloud.BootImageSpec           `json:"bootImageSpec"`
+	DefaultInstanceType    string                        `json:"defaultInstanceType"`
+	DefaultVolumeSize      string                        `json:"defaultVolumeSize"`
+	StandbyCells           []nodemanager.StandbyNodeSpec `json:"standbyCells"`
+	CloudInitFile          string                        `json:"cloudInitFile"`
+	Itzo                   ItzoConfig                    `json:"itzo"`
+	ExtraCIDRs             []string                      `json:"extraCIDRs"`
+	ExtraSecurityGroups    []string                      `json:"extraSecurityGroups"`
+	Nametag                string                        `json:"nametag"`
+	StatusInterval         int                           `json:"statusInterval"`
+	HealthCheck            HealthCheckConfig             `json:"healthcheck"`
+	PrivateIPOnly          *bool                         `json:"privateIPOnly"`
+	CellConfig             map[string]string             `json:"cellConfig"`
+	DefaultIAMPermissions  string                        `json:"defaultIAMPermissions"`
+	UseCloudParameterStore bool                          `json:"useCloudParamaterStore"`
```

Comment:
could you this param with short explanation to provider docs as well?

Reply:
Good point :+1: